temporian/implementation/numpy/operators/window/base.py

```diff
@@ -90,3 +90,16 @@ def _compute(
                 kwargs["sampling_timestamps"] = sampling_timestamps
             dst_feature = implementation(**kwargs)
             dst_features.append(dst_feature)
+
+    def apply_feature_wise(
```

Comment:
This looks like it can be merged with `_compute()` above

Reply:
Almost.

- It would mean a new implementation class is instantiate for each feature (probably okay).
- "apply_feature_wise" does not support sampling. For the Beam implementation, sampling is a bit more complex as it requires two different streams of data.